Makefile

```diff
@@ -188,10 +188,6 @@ apidoc_dirs = out/doc out/doc/api/ out/doc/api/assets
 
 apiassets = $(subst api_assets,api/assets,$(addprefix out/,$(wildcard doc/api_assets/*)))
 
-website_files = \
-	out/doc/sh_main.js    \
-	out/doc/sh_javascript.min.js
-
```

Comment:
shouldn't the path on these files change, instead of deleting the variable entirely?